src/tests/process/dhcpv6/Request

```diff
@@ -0,0 +1,32 @@
+subrequest DHCPv6.Request {
+	&Transaction-ID = 0x1e291d
+	&Client-ID.DUID = LL
+
+	&Client-ID.DUID.LL.Hardware-Type = Ethernet
+	&Client-ID.DUID.LL.Hardware-Type.Ethernet.Address = 00:01:02:03:04:05
+
+	&Server-ID.DUID = LLT
+	&Server-ID.DUID.LLT.Hardware-Type = Ethernet
+	&Server-ID.DUID.LLT.Time = "Nov 21 2012 08:36:00 UTC"
+	&Server-ID.DUID.LLT.Hardware-Type.Ethernet.Address = 00:11:22:33:44:55
+
+	&Option-Request = DNS-Servers
+	&Option-Request = AFTR-Name
+
+	&Elapsed-Time = 0
+
+	&IA-PD.IAID = 33752069
+	&IA-PD.T1 = 3600
+	&IA-PD.T2 = 5400
+	&IA-PD.Options = {
+		&IA-PD-Prefix.Preferred-Lifetime = 7200
+		&IA-PD-Prefix.Valid-Lifetime = 7500
+		&IA-PD-Prefix.IPv6-Prefix = 2a00:1:1:100::/56
+	}
+
+	call dhcpv6 {
+		if (&reply.Packet-Type != Reply) {
+			test_fail
+		}
+	}
```

Comment:
You need to check that Client-ID and Server-ID are echo'd back from the server in the reply.